doc/lua/sync_async.md

```diff
@@ -0,0 +1,654 @@
+---
+layout: doc_lua
+title: Sync and Async API comparison
+---
+
+{::options parse_block_html="true" /}
+
+<div id="toc">
+  * this unordered seed list will be replaced by toc as unordered list
+  {:toc}
+</div>
+
+## Overview {#overview}
+
+Prior to 1.8.0 if you needed to perform an action involving network request
+(i.e. Redis query, Anti-virus scan), you had to use callback-style approach.
+You define callback and initiate an asynchronous request and stop the execution
+to let other tasks proceed. 
+
+As soon as request is done the call back is called and the operation continues.
```

Comment:
Comma after `done`, or, even better "As soon as the request is completed,"; "callback". It is not clear what do you mean by "operation continues", it seems to be redundant.